docs/manual-setup.adoc

```diff
@@ -215,6 +219,16 @@ Confirm all of the transactions and verify that the TBTC has left your wallet. Y
   (top right -> Settings -> Advanced -> Reset Account)
 * During the keep-ecdsa phase of running install.sh it may complain about sha3.
   If that happens, cd into keep-ecdsa and run: `$ npm install --save-dev sha3`
+* If you see an error coming from npm complaining about dependencies (e.g. babel-cli),
+  try deleting package-lock.json (e.g. if npm cannot find file
+  local-setup/tbtc/solidity/node_modules/.staging/@keep-network/keep-ecdsa-64660964/node_modules/@babel/cli,
+  delete package-lock.json file located in local-setup/tbtc/solidity) and run
```

Comment:
Let's use backticks for `package-lock.json` and `local-setup/tbtc/solidity`, and `local-setup/tbtc/solidity/node_modules/.staging/@keep-network/keep-ecdsa-64660964/node_modules/@babel/cli` - that's how we format file names in earlier sections of this doc.

Reply:
I added it here:
https://github.com/keep-network/local-setup/pull/68